pkg/cache/context.go

```diff
@@ -434,6 +458,19 @@ func (ctx *Context) AddApplication(request *interfaces.AddApplicationRequest) in
 
 	ctx.lock.Lock()
 	defer ctx.lock.Unlock()
+
+	// add resource quota info as a app tag
+	log.Logger.Info("### retrieve namespace info 1")
+	if ns, ok := request.Metadata.Tags[common.AppTagNamespace]; ok {
+		log.Logger.Info("### retrieve namespace info 2", zap.String("namespace", ns))
+		resourceQuota := ctx.getNamespaceResourceQuota(ns)
+		if resourceQuota != nil && !common.IsZero(resourceQuota) {
```

Comment:
This does not capture the case that we set `mem=0` and `cpu=10` but we cannot apply that on the core side as it would stop scheduling for that namespace.
I think we need to catch this earlier than the core side.